fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/ExpressionUtils.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+
+import com.ibm.fhir.database.utils.query.Alias;
+import com.ibm.fhir.database.utils.query.Select;
+import com.ibm.fhir.database.utils.query.WhereFragment;
+import com.ibm.fhir.database.utils.query.node.ACosExpNode;
+import com.ibm.fhir.database.utils.query.node.BigDecimalBindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.BindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.CosExpNode;
+import com.ibm.fhir.database.utils.query.node.DoubleBindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.ExpNode;
+import com.ibm.fhir.database.utils.query.node.InstantBindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.IntegerBindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.LongBindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.SinExpNode;
+import com.ibm.fhir.database.utils.query.node.StringBindMarkerNode;
+
+/**
+ * Collection of utility functions for building predicate expressions. Consumers
+ * should static import the functions they require for cleaner syntax.
+ */
+public class ExpressionUtils {
```

Comment:
add  a private constructor to present misuse

Reply:
ok